src/discord/cmd/ping.rs

```diff
@@ -0,0 +1,28 @@
+use crate::discord::cmd::CommandExecutable;
+use crate::discord::error::Error;
+use serenity::async_trait;
+use serenity::client::Context;
+use serenity::model::application::interaction::application_command::ApplicationCommandInteraction;
+use serenity::model::application::interaction::{Interaction, InteractionResponseType};
+
+pub struct PingCmd {}
+
+/// Execute the "ping" command.
+#[async_trait]
+impl CommandExecutable for PingCmd {
+    async fn execute(
+        &self,
+        ctx: &Context,
+        _: &Interaction,
+        command: &ApplicationCommandInteraction,
+    ) -> Result<(), Error> {
+        command
+            .create_interaction_response(&ctx.http, |response| {
```

Comment:
Same here, epheremerals are usefull for reply that don't need to be showed to others users :)

Reply:
Same answer as for the other comment. Thx ☺️